src/interfaces/address-interface.ts

```diff
@@ -0,0 +1,31 @@
+/** Interface to define a base address. */
```

Comment:
Maybe pluralize this file name so we can tell there are multiple IFs in here.

Reply:
k, will do

Reply:
Updated 👍